src/main/resources/com/puppycrawl/tools/checkstyle/grammar/java.g

```diff
@@ -1250,6 +1274,36 @@ caseSList
         {#caseSList = #(#[SLIST,"SLIST"],#caseSList);}
     ;
 
+switchRule
+    :   (       (switchLabeledExpression)=>     se:switchLabeledExpression
+        |       (switchLabeledBlock)=>      sb:switchLabeledBlock
+        |       (switchLabeledThrow)=>    st:switchLabeledThrow
```

Comment:
as we do this game of vertical code alignment: 
```
    :   (       (switchLabeledExpression)=>  se:switchLabeledExpression
        |       (switchLabeledBlock)=>           sb:switchLabeledBlock
        |       (switchLabeledThrow)=>          st:switchLabeledThrow
```

Reply:
Fixed.